entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
I think there are no bashisms, so this could work even if switching base containers.
```suggestion
#!/bin/sh
```